File: C#Integration/PokerProtocol/Commands/Game/PlayerPlayMoneyCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using EricUtility;
using PokerProtocol.Commands.Lobby.Response;
using PokerWorld.Game;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Game
{
    public class PlayerPlayMoneyCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return PlayerPlayMoneyCommand.COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "gamePLAY_MONEY";

        private readonly int m_Played;

        public int Played
        {
            get { return m_Played; }
        }

        public PlayerPlayMoneyCommand(StringTokenizer argsToken)
        {
            m_Played = int.Parse(argsToken.NextToken());
        }

        public PlayerPlayMoneyCommand(int played)
        {
            m_Played = played;
        }

        public override void Encode(StringBuilder sb)
        {
            Append(sb, m_Played);
        }
    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands2/game/PlayerPlayMoneyCommand.java
package bluffinmuffin.protocol.commands2.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands2.AbstractCommand;

public class PlayerPlayMoneyCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return PlayerPlayMoneyCommand.COMMAND_NAME;
    }
    
    private final int m_played;
    
    public static String COMMAND_NAME = "gamePLAY_MONEY";
    
    public PlayerPlayMoneyCommand(StringTokenizer argsToken)
    {
        m_played = Integer.parseInt(argsToken.nextToken());
    }
    
    public PlayerPlayMoneyCommand(int played)
    {
        m_played = played;
    }
    
    @Override
    public void encode(StringBuilder sb)
    {
        append(sb, m_played);
    }
    
    public int getPlayed()
    {
        return m_played;
    }
}
